pkg/artifactory/resource_artifactory_ldap_setting.go

```diff
@@ -0,0 +1,280 @@
+package artifactory
+
+import (
+	"context"
+	"encoding/xml"
+	"github.com/go-resty/resty/v2"
```

Comment:
This line should be part of the group below.

Reply:
I think if he just applies `gofmt` it will do that job

Reply:
Our `make fmt` runs `goimports` as well, which is what reorg the statements.